src/nxdoc/nuxeo-add-ons/nuxeo-aspera-connector.md

```diff
@@ -176,19 +551,19 @@ aspera.download.acess.key.id=
 aspera.download.acess.key.secret=
 ```
 
-## Functional Overview
+### Functional Overview
 
 The Nuxeo Aspera Connector uses "Transfers" to add content to the system.
 
 The process is divided into 2 steps: Upload and Create.
 
-In the first step, the user **uploads** content to the transfer bucket. While in this step, he can add files to the bucket and add properties that will apply to the content. Properties can be added to all files at once (common metadata), to multiple files at once (bulk edit of metadata) and one at a time.
+In the first step, the user **uploads** content to the transfer bucket. While in this step, a user can add content to the bucket and add properties that will apply to the content. Properties can be added to all files at once (common metadata), to multiple files at once (bulk edit of metadata) and one at a time.
```

Comment:
This version only allows files, right? Not sure if using `content` is clear enough since we use it above with a different meaning 🤔

Reply:
yes, thanks.  I realized i got a little "find" edit happy within my doc pages after I got to a place where I saw a typo and said "I could swear I fixed that" and then i realized i was down in the 2.0.x area.  I thought I backed out my changes.  wil review. 👍